### PR TITLE
Tags/redesign

### PR DESCRIPTION
**Pull request by TruddyTheDuddi**
**What**
Worked on redesigning the Tags repository

- Tags repo `fragment_tags_display.xml` looks clean and can be well integrated in the different views across the app. Redesigned the button to click for adding. Made it more user friendly and intuitive by adding a placeholder text, if no tags are selected, which can also be clicked to add the first new tags.
- Changed a few things in `TagsDisplayFragment.kt` so that it works. Also removed padding settings because it had no effect on the tags. But not fully tested yet

### CODE REVIEW

**Review by atappy — APPROVED** [review]
a really good job, looks nice !

File: app/src/main/java/com/github/palFinderTeam/palfinder/tag/Category.kt [hunk 1]
```diff
@@ -9,18 +9,18 @@ import kotlinx.serialization.Serializable
...
-    DUMMY_TAG6("dummy tag 6"),
-    DUMMY_TAG7("dummy tag 7"),
-    DUMMY_TAG8("dummy tag 8"),
-    DUMMY_TAG9("dummy tag 9"),
-    DUMMY_TAG10("dummy tag 10"),
-    DUMMY_TAG11("dummy tag 11"),
+    DRINKING("Drinking"),
+    WORKING_OUT("Working out"),
+    CINEMA("Cinema"),
+    DUMMY_TAG1("Walking/Hiking"),
```
[the remark this answers is not in the document]
Comment (TruddyTheDuddi) [reply app/src/main/java/com/github/palFinderTeam/palfinder/tag/Category.kt:15]: My bad! I changed the string but not the identifier itself!

File: app/src/main/res/drawable/ic_baseline_new_label_24.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:tint="?attr/colorControlNormal">
```
Comment (Gerussi) [inline app/src/main/res/drawable/ic_baseline_new_label_24.xml:6]: 👍

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -3,4 +3,7 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+
+    <dimen name="tags_padding_vertical">5</dimen>
```
Comment (Gerussi) [inline app/src/main/res/values/dimens.xml:7]: I think you should put the metric for dimen, if you want to only have integer, you can use the integers.xml file

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -3,4 +3,7 @@
...
+    <dimen name="tags_padding_horizontal">10</dimen>
```
Comment (Gerussi) [inline app/src/main/res/values/dimens.xml:8]: I think you should put the metric for dimen, if you want to only have integer, you can use the integers.xml file

**Review by Gerussi — APPROVED** [review]
Very good! Please just adressed the dimen item without the metrics, thanks!

File: app/src/main/res/values/dimens.xml (hunk 4 above)
Comment (TruddyTheDuddi) [reply app/src/main/res/values/dimens.xml:8]: I didn't use it, I removed it

**Pull request merged**